samples/apps/copilot-chat-app/webapp/src/redux/features/app/AppState.ts

```diff
@@ -3,6 +3,13 @@ import { AlertType } from '../../../libs/models/AlertType';
 // Copyright (c) Microsoft. All rights reserved.
 export interface AppState {
     alerts?: Alerts;
+    loggedInUserInfo?: LoggedInUserInfo;
+}
+
+export interface LoggedInUserInfo {
+    id: string;
+    email: string;
+    fullName: string;
```

Comment:
nit: two rename suggestions:
- `LoggedInUserInfo` to `ActiveUserInfo` since MSAL supports multiple logged in users at one time
- `fullName` to `username` since it can fallback onto email when user's actual name is available